src/stories/Library/search-result-page/FacetLineSelected.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC } from "react";
+import { Tag } from "../tag/Tag";
+import { FacetLineItem } from "./FacetLine";
+
+export interface FacetLineSelectedProps {
+  items: FacetLineItem<"term">[];
+}
+
+const FacetLineSelected: FC<FacetLineSelectedProps> = ({ items }) => {
+  return (
+    <ul className="facet-line-selected">
+      {items.map(({ title }) => (
+        <li className="facet-line-selected__item">
+          <Tag showCloseIcon hasBackground>
```

Comment:
These "selected" tags should not have outline according to the design in Figma. Please adjust accordingly.
![image](https://user-images.githubusercontent.com/28546954/199273476-fcf9424f-afe9-40c0-bbe9-c5d6f65bd5c7.png)

Reply:
Ok I asked Kasper about that and he told me otherwise. I'll look into that.

Reply:
I have changed it, but to be honest I would like Kasper to look at the whole Tag setup, he created those classes originally. I just cleaned up the component logic.